charts/dify/templates/config.tpl

```diff
@@ -260,6 +273,16 @@ MILVUS_PORT: {{ .Values.externalMilvus.port | toString | quote }}
 # MILVUS_PASSWORD: {{ .Values.externalMilvus.password | quote }}
 # The milvus tls switch.
 MILVUS_SECURE: {{ .Values.externalMilvus.useTLS | toString | quote }}
+{{- else if .Values.externalTencent.enabled}}
```

Comment:
We'll need a canonical name for this vector database instead of  `Tencent` which suggests the name of the enterprise itself